src/components/ReportActionItem/ReportPreview.js

```diff
@@ -142,7 +142,7 @@ const ReportPreview = (props) => {
                     iouReport={props.iouReport}
                     onPress={(paymentType) => IOU.payMoneyRequest(paymentType, props.chatReport, props.iouReport)}
                     enablePaymentsRoute={ROUTES.BANK_ACCOUNT_NEW}
-                    addBankAccountRoute={ROUTES.IOU_DETAILS_ADD_BANK_ACCOUNT}
+                    addBankAccountRoute={ROUTES.IOU_SEND_ADD_BANK_ACCOUNT}
```

Comment:
Same here, let's use `getBankAccountRoute` and pass the policyID if this is a `policyExpenseChat`